code/game/objects/structures/stool_bed_chair_nest/stools.dm

```diff
@@ -30,7 +32,10 @@
 /obj/structure/stool/attackby(obj/item/weapon/W as obj, mob/user as mob, params)
 	if(istype(W, /obj/item/weapon/wrench))
 		playsound(loc, W.usesound, 50, 1)
-		new /obj/item/stack/sheet/metal(loc)
+		if(is_wooden)

```

Comment:
I was trying to use a ternary operator, but somehow, probably due to lack of sleep, couldn't make it work in DM... or this'd be a single line, not 4...